app/views/dish/new.html.haml

```diff
@@ -0,0 +1,18 @@
+.row
+  .medium-8.medium-centered
+    %h2 Add new dish to '#{@menu.name}' menu
+    - if @dish.errors.any?
+      .callout.small.alert
+        %ul
+        - @dish.errors.full_messages.each do |error|
+          %li
+            = error
```

Comment:
...as I see that you use here... Okay. Still if feels a bit redundant to have both a flash and the error messages displayed. Not a blocker though.